bundles/test/org.eclipse.smarthome.extension.sample/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Sample Extension Service
+Bundle-SymbolicName: org.eclipse.smarthome.extension.sample
+Bundle-Version: 0.8.0.qualifier
+Export-Package: org.eclipse.smarthome.extension.sample.internal
+Import-Package: org.apache.commons.lang;version="2.6.0",
```

Comment:
I am fine with version that version constraints if we know, that at least that version is necessary.
But the coding guidelines stated that the usage must be used if thoughtfully added, only.
Mistake or thoughtfully added?